mitodl/open-discussions
# Added Rest APIs to add/remove contributors in a channel

#### What are the relevant tickets?
closes #132 

#### What's this PR do?
Adds a REST API to add and remove contributors in a channel

#### How should this be manually tested?
if you disable the `JwtIsStaffOrReadonlyPermission` in the views, you can use the django rest framework UI to perform the requests and you can go to `http://reddit.local/r/<channel_name>/about/contributors/` and verify that the action you perform via rest API correspond to what you see in the UI
